doc/classes/AStar.xml

```diff
@@ -16,6 +17,20 @@
 
 		    func _estimate_cost(u, v):
 		        return min(0, abs(u - v) - 1)
+		[/gdscript]
+		[csharp]
+		public class MyAStar : AStar
+		{
+		    public override float _ComputeCost(int u, int v)
+		    {
+		        return abs(u - v)
```

Comment:
This is not valid, `abs` does not exist globally, you have to use `Mathf` for math methods. Also, you are missing a semicolon. Also, the return type should match the argument type.

```suggestion
		    public override int _ComputeCost(int u, int v)
		    {
		        return Mathf.Abs(u - v);
```

Please check your PR for similar problems (there is at least one right below this).

Reply:
1. Fixed and added to translation tool
2. Fixed and added to translation tool
3. This has to be fixed in the Godot API. `_compute_cost` and `_estimate_cost` are both virtual function that require a float value.